docs/docs/monitoring/analytics/example-queries.mdx

```diff
@@ -0,0 +1,200 @@
+</figure>
+
+## Number of sessions per channel
+
+If you're connecting your assistant to multiple channels, it could be useful to look at
+the number of sessions per channel, let's say per week. The query you would need for this metric is:
+
+```sql
+SELECT
+  "public"."rasa_user_message"."channel" AS "channel",
+  "public"."rasa_user_message"."timestamp" AS "timestamp",
+  count(distinct "public"."rasa_user_message"."session_id") AS "count"
+FROM "public"."rasa_user_message"
+GROUP BY 1, 2
+ORDER BY 1 ASC, 2 ASC
+```
+
+
+<figure align="center">
+  <img
+    alt="Number of sessions per channel visualised in Metabase."
+    src={useBaseUrl("/img/analytics/graph-number-sessions-channel.png")}
+    width="100%"
+  />
+  <figcaption>Number of sessions per channel visualised in Metabase.</figcaption>
+</figure>
+
+
+## Top N intents
+
+To improve your assistant, you could look into the variety of intents your users express.
+The query below selects the top 5 intents which could help you have a good perspective on that topic:
+
+```sql
+SELECT
+  "public"."rasa_user_message"."intent" AS "intent",
+  count(*) AS "count"
+FROM "public"."rasa_user_message"
+GROUP BY 1
+ORDER BY 2 DESC, 1 ASC
+LIMIT 5
+```
+
+
+<figure align="center">
+  <img
+    alt="Top 5 intents visualised in Metabase."
+    src={useBaseUrl("/img/analytics/graph-top-5-intents.png")}
+    width="100%"
+  />
+  <figcaption>Top 5 intents visualised in Metabase.</figcaption>
+</figure>
+
+Moreover, you can look for the intent distribution over time:
+
+```sql
+SELECT
+  "public"."rasa_user_message"."intent" AS "intent",
+  date_trunc('month', "public"."rasa_user_message"."timestamp") AS "timestamp",
+  count(*) AS "count" FROM "public"."rasa_user_message"
+GROUP BY 1, 2
+ORDER BY 1 ASC, 2 ASC
+```
+
+<figure align="center">
+  <img
+    alt="Intent distribution over time visualised in Metabase."
+    src={useBaseUrl("/img/analytics/graph-intent-distribution.png")}
+    width="100%"
+  />
+  <figcaption>Intent distribution over time visualised in Metabase.</figcaption>
+</figure>
+
+
+## Escalation rate
+
+Escalation rate or human hand-off rate is a measure of how many conversations does the assistant pass to a human agent.
+This metric can help you gain a better understanding of what happens during a conversation.
+Let's say you have an intent named `handoff_to_support`. You'll get the escalation rate over time
+with this sample query:
+
+```sql
+WITH "sessions" AS (
+    SELECT
+        "public"."rasa_user_message"."session_id" AS "session_id",
+        date_trunc('month', "public"."rasa_user_message"."timestamp") AS "timestamp",
+        (
+          CASE "public"."rasa_user_message"."intent"
+            WHEN 'handoff_to_support'
+            THEN 1 ELSE 0
+          END
+        ) AS "has_handoff_to_support"
+    FROM "public"."rasa_user_message"
+),
+"sessions_with_handoff" AS (
+    SELECT
+      "session_id",
+      "timestamp",
+      SUM(
+        CASE
```

Comment:
do we need the case here? isn't `has_handoff_to_support` set to `0` or `1`?

Reply:
i think you did it differently for the abandonment one

Reply:
the thing is that for some sessions there was multiple "out_of_scope" intents. But yeah you're right I'll fix that